core/common/src/test/java/alluxio/master/PollingMasterInquireClientTest.java

```diff
@@ -51,4 +63,39 @@ public void pollRejectingDoesntHang() throws Exception {
       // Expected
     }
   }
+
+  @Test(timeout = 10000)
+  public void testGetPrimaryRpcAddress() throws IOException {
```

Comment:
nit: no need for "test" key word in the test method name because you already tagged it